src/utils/sample.jsx

```diff
@@ -2626,3 +2626,205 @@ export const tableData = [
     },
   },
 ];
+
+export const barChartData = {
```

Comment:
It would be nice to get this file split up into different categories / sections. 2630 lines of sample data is difficult to navigate if someone is looking for examples. Maybe all of the "chartable data" in their own file?

Reply:
done